.github/workflows/tools.yml

```diff
@@ -74,11 +74,16 @@ jobs:
       if: steps.changes.outputs.instance_validator == 'true'
       run: |
         pylint --rcfile ./styles/pylintrc ./tools/validators/instance_validator
-    - name: Instance Yaml Validator Tests
+    - name: Instance Yaml Validator Unit Tests
       if: steps.changes.outputs.instance_validator == 'true' || steps.changes.outputs.type_validator == 'true'
       run: |
        python -m unittest discover -p '*_test.py' --failfast -v
       working-directory: ./tools/validators/instance_validator
+    - name: Instance Yaml Validator Integraion Test
```

Comment:
interesting edge case for guid, the file used here contains the building guid as `guid123` I think we need to add a guid check to be conform to a uuid4. ok to do this a seperate pr